package.json

```diff
@@ -35,50 +35,51 @@
     "icons:update": "node ./packages/icons/update-icons.js"
   },
   "dependencies": {
-    "@angular/animations": "^16.1.3",
-    "@angular/cdk": "^16.1.3",
-    "@angular/common": "^16.1.3",
-    "@angular/compiler": "^16.1.3",
-    "@angular/core": "^16.1.3",
-    "@angular/forms": "^16.1.3",
-    "@angular/platform-browser": "^16.1.3",
-    "@angular/platform-browser-dynamic": "^16.1.3",
-    "@angular/router": "^16.1.3",
+    "@angular/animations": "^17.0.2",
+    "@angular/cdk": "^17.0.0",
+    "@angular/common": "^17.0.2",
+    "@angular/compiler": "^17.0.2",
+    "@angular/core": "^17.0.2",
+    "@angular/forms": "^17.0.2",
+    "@angular/platform-browser": "^17.0.2",
+    "@angular/platform-browser-dynamic": "^17.0.2",
+    "@angular/router": "^17.0.2",
     "@ngx-formly/core": "^6.1.5",
     "dompurify": "^3.0.3",
     "normalize.css": "^8.0.1",
     "rxjs": "^7.8.1",
-    "tslib": "^2.5.0",
-    "zone.js": "~0.13.0"
+    "tslib": "^2.6.2",
+    "zone.js": "~0.14.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^16.1.3",
-    "@angular-devkit/schematics": "^16.1.3",
-    "@angular-eslint/builder": "^16.0.3",
-    "@angular-eslint/eslint-plugin": "^16.0.3",
-    "@angular-eslint/eslint-plugin-template": "^16.0.3",
-    "@angular-eslint/schematics": "^16.0.3",
-    "@angular-eslint/template-parser": "^16.0.3",
-    "@angular/cli": "^16.1.3",
-    "@angular/compiler-cli": "^16.1.3",
-    "@compodoc/compodoc": "^1.1.21",
-    "@ngneat/spectator": "^14.0.0",
-    "@storybook/addon-a11y": "^7.4.0",
+    "@angular-devkit/build-angular": "^17.0.0",
+    "@angular-devkit/schematics": "^17.0.0",
+    "@angular-eslint/builder": "^17.0.1",
+    "@angular-eslint/eslint-plugin": "^17.0.1",
+    "@angular-eslint/eslint-plugin-template": "^17.0.1",
+    "@angular-eslint/schematics": "^17.0.1",
+    "@angular-eslint/template-parser": "^17.0.1",
+    "@angular/cli": "^17.0.0",
+    "@angular/compiler-cli": "^17.0.2",
+    "@compodoc/compodoc": "^1.1.22",
+    "@ngneat/spectator": "^15.0.1",
+    "@storybook/addon-a11y": "^7.5.3",
     "@storybook/addon-docs": "^7.5.1",
-    "@storybook/addon-essentials": "^7.4.0",
+    "@storybook/addon-essentials": "^7.5.3",
+    "@storybook/addon-interactions": "^7.5.3",
     "@storybook/addon-interactions": "^7.5.1",
-    "@storybook/addon-mdx-gfm": "^7.4.0",
-    "@storybook/angular": "^7.4.0",
-    "@storybook/test-runner": "^0.11.0",
-    "@testing-library/angular": "^14.1.1",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/user-event": "^14.4.3",
+    "@storybook/addon-mdx-gfm": "^7.5.3",
+    "@storybook/angular": "^7.5.3",
+    "@storybook/test-runner": "^0.14.0",
+    "@testing-library/angular": "^14.4.0",
+    "@testing-library/jest-dom": "^6.1.4",
+    "@testing-library/user-event": "^14.5.1",
     "@types/dompurify": "^3.0.2",
     "@types/jest": "^29.5.1",
     "@types/jest-axe": "^3.5.5",
     "@types/node": "^18.11.10",
```

Comment:
Not very important but this should be bumped to 18.17

Reply:
I'll do this in another PR to not stale previous approves ^^